app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    name_file = command.split(" ")
```

Comment:
The `split` method should specify a delimiter to split the command string. Currently, it defaults to splitting by any whitespace, which might not be the intended behavior if there are extra spaces.